activation/validation_test.go

```diff
@@ -65,7 +65,7 @@ func Test_Validation_VRFNonce(t *testing.T) {
 	t.Run("wrong commitmentAtxId", func(t *testing.T) {
 		t.Parallel()
 
-		commitmentAtxId := types.RandomATXID()
```

Comment:
This test failed in a CI run. The commitmentAtx being random gives a small chance of coincidentally passing the VRFNonce check.

Set it to a fixed value to avoid that in the future.